catalog/MDCCatalog/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>70.0.0</string>
+	<string>70.1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
```

Comment:
Should the CFBundleVersion value be correct as well?

Reply:
I'm not sure - it looks like we haven't been updating it. Should we?